model/src/main/java/org/cbioportal/genome_nexus/model/VUEs.java

```diff
@@ -0,0 +1,88 @@
+package org.cbioportal.genome_nexus.model;
+
+import java.util.List;
+
+public class VUEs {
+
+    private String hugoGeneSymbol;
+    private String transcriptId;
+    private String genomicLocation;
+    private int[] pubmedIds;     
+    private String context;
+    private String comment;
+    private String referenceText;
+    private String defaultEffect;          
+    private List<RevisedProteinEffect> revisedProteinEffects;
+
+    public String gethugoGeneSymbol() {
```

Comment:
```suggestion
    public String getHugoGeneSymbol() {
```
As well as all getter and setter functions in this file. Could simply delete all and re-generate from vscode